app/server.go

```diff
@@ -224,7 +224,7 @@ func NewServer(options ...Option) (*Server, error) {
 	}
 
 	// This is called after initLogging() to avoid a race condition.
-	mlog.Info("Server is initializing...")
+	mlog.Info("Server is initializing...", mlog.String(" Go version", runtime.Version()))
```

Comment:
```suggestion
	mlog.Info("Server is initializing...", mlog.String("go_version", runtime.Version()))
```

This would be more consistent with how we use the mlog fields. First string is similar to a variable name and then the actual value.